pikaur/prompt.py

```diff
@@ -1,26 +1,76 @@
 import sys
+import tty
 from typing import Callable, List
 
+from pikaur.config import PikaurConfig
+
+from pikaur.args import PikaurArgs
```

Comment:
i do prefer relative imports

Reply:
Did not know that was a thing you could do to be honest, changed those.